dll/cpl/sysdm/smbios.c

```diff
@@ -343,6 +416,29 @@ void AppendSystemFamily(PWSTR pBuf, SIZE_T cchBuf, PCHAR * DmiStrings, PWSTR dev
     }
 }
 
+static
+BOOL TrimNonAscii(PCHAR DmiString)
```

Comment:
Technically speaking, the function name is misleading, since it refuses anything below 0x20, which are ASCII chars.
Is that why you don't use isascii() ?

Reply:
Right... let's name it `TrimNonPrintable` then.

(suggestions are welcome) :grin:

Reply:
Then I also wonder whether you could also use one of these functions: http://www.cplusplus.com/reference/cctype/isprint/ or another one from the list at http://www.cplusplus.com/reference/cctype/ .

Reply:
@HBelusca I tested `isprint` - it works fine here, but I think checking the characters directly is faster than calling a function on each loop iteration.

Renamed the function.